packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -613,8 +629,17 @@ class _TableRowState<T> extends State<TableRow<T>>
     }
   }
 
+  TextStyle _fixedFontStyle(BuildContext context) {
```

Comment:
nit:  consider moving this helper to src/flutter/theme.dart

Reply:
done